dashboard/app/models/game.rb

```diff
@@ -148,6 +149,10 @@ def self.standalone_video
     @@game_standalone_video ||= find_by_name("StandaloneVideo")
   end
 
+  def self.standalone_video2
```

Comment:
I tried for a while, and am open to ideas.  I thought about `video` (but then the model collides with an existing `Video` class in the sashboard), `VideoLab` (since it has characteristics of a lab, but it isn't really a lab), `VideoLevel` (but then naming of related files gets weird)` for starters.  This really will look a lot like a `standalone_video` level, but it will be React-based with a proper JS app backing it, and with support for level changes without page reloads.

Reply:
After seeing how much server-side support we have built for `StandaloneVideo`, including levelbuilder support, and our discussion about the disadvantages of adding another model and game type, I'm trying a different approach: we'll use `StandaloneVideo` on the server, but when we see it in use alongside Music Lab levels, we'll use a newer client-side implementation (name to be determined) which is React-based and supports changing levels without reloads.